frontend/benefit/applicant/src/components/cookieConsent/CookieConsent.tsx

```diff
@@ -0,0 +1,88 @@
+import useLocale from 'benefit/applicant/hooks/useLocale';
+import { BackendEndpoint } from 'benefit-shared/backend-api/backend-api';
+import { CookieModal } from 'hds-react';
+import { useRouter } from 'next/router';
+import React from 'react';
+import useBackendAPI from 'shared/hooks/useBackendAPI';
+
+const CookieConsent: React.FC = () => {
+  const locale = useLocale();
+  const router = useRouter();
+  const { axios } = useBackendAPI();
+  const { pathname, asPath, query } = router;
+
+  const onLanguageChange = (newLanguage: 'fi' | 'sv' | 'en'): void => {
+    void axios.get(BackendEndpoint.USER_OPTIONS, {
+      params: { lang: newLanguage },
+    });
+
+    void router.push({ pathname, query }, asPath, {
+      locale: newLanguage,
+    });
+  };
+
+  const askemHostname = 'reactandshare.com';
+
+  type CookieTexts = {
+    title: { fi: string; sv: string; en: string };
+    askemDescription: { fi: string; sv: string; en: string };
+  };
+
+  const text: CookieTexts = {
```

Comment:
Shouldn't these be in locale JSON files?

Reply:
Definitely yes, but just could not make it work. The HDS cookie banner has some own localizations that mess up with ours. Saw same kind of a solution here <https://github.com/City-of-Helsinki/employment-services-ui/blob/08cbb83303ca96094d60b4cbb0d4678010b0fc12/src/hooks/useAnalytics.ts#L33>

Reply:
Hmm... suppouse you can try importing json as such and just stick the objects in like I've done in tests.

Reply:
For example, 

https://github.com/City-of-Helsinki/yjdh/blob/main/frontend/benefit/applicant/browser-tests/page-model/deminimis.ts#L3

and

https://github.com/City-of-Helsinki/yjdh/blob/main/frontend/benefit/applicant/browser-tests/page-model/deminimis.ts#L137